clients/src/main/java/org/apache/kafka/clients/consumer/internals/AbstractCoordinator.java

```diff
@@ -418,6 +415,7 @@ public void handle(SyncGroupResponse syncResponse,
                            RequestFuture<ByteBuffer> future) {
             Errors error = Errors.forCode(syncResponse.errorCode());
             if (error == Errors.NONE) {
+                log.info("Successfully joined group {} with generation {}", groupId, generation);
                 log.debug("Received successful sync group response for group {}: {}", groupId, syncResponse.toStruct());
```

Comment:
I was debating on it. Maybe there's some useful information to pull out of the sync response, but probably not if it was successful anyway. Perhaps we just remove it?